templates/channel.pug

```diff
@@ -46,7 +46,7 @@ html(lang="en")
               #userlist
               #messagebuffer.linewrap
               form(action="javascript:void(0)")
-                input#chatline.form-control(type="text", maxlength="320", style="display: none")
+                input#chatline.form-control(type="text", maxlength=`${maxMsgLen}`, style="display: none")
```

Comment:
Since `maxMsgLen` is being passed as a template parameter, you should be able to just do `maxlength=maxMsgLen` and not need the extra `` `${}` ``

Reply:
I didn't try, I wasn't sure if it would automatically place it as a string or not if I did that. Will check.

Reply:
Neat, that does work. I figured it would place it as a number and break things.